### PR TITLE
Add error handle for webhook topics

### PR DESCRIPTION
**Pull request by aarthi-axim**
## Status
**READY**

## Description
Added error handle to webhook topics.

### CODE REVIEW

File: lib/dynamic_screening_solutions/api.py [hunk 1]
```diff
@@ -458,6 +458,9 @@ def get_webhook_topics(self, company_id):
         except Exception:
             webhook_topics = []
 
+        if 'error' in webhook_topics and webhook_topics['error'] is True:
+            webhook_topics = []
+
```
Comment (jontsai) [inline lib/dynamic_screening_solutions/api.py:463]: Let's move these 3 lines into the `try` block

**Review by jontsai — CHANGES_REQUESTED** [review]

File: lib/dynamic_screening_solutions/api.py [hunk 2]
```diff
@@ -458,6 +458,9 @@ def get_webhook_topics(self, company_id):
         except Exception:
             webhook_topics = []
 
+        if 'error' in webhook_topics and webhook_topics['error'] is True:
+            webhook_topics = []
```
Comment (jontsai) [inline lib/dynamic_screening_solutions/api.py:462]:
```suggestion
            if webhook_topics.get('error'):
                webhook_topics = []
        except Exception:
            webhook_topics = []

```

We can move it into the `try` block and simply the expression to just do a `.get`

**Review by jontsai — CHANGES_REQUESTED** [review]

File: lib/dynamic_screening_solutions/api.py (hunk 2 above)
Comment (aarthi-axim) [reply lib/dynamic_screening_solutions/api.py]: Done

**Review by jontsai — APPROVED** [review]
LGTM

**Pull request merged**